alpha-solver/src/test/java/at/ac/tuwien/kr/alpha/regressiontests/SolverStatisticsTests.java

```diff
@@ -51,32 +54,33 @@ public void setUp() {
 	}
 
 	@RegressionTest
-	public void checkStatsStringZeroChoices(RegressionTestConfig cfg) {
+	public void checkStatsStringZeroChoices(SystemConfig cfg) {
 		Solver solver = buildSolverForRegressionTest("a.", cfg);
 		assumeTrue(solver instanceof StatisticsReportingSolver);
 		collectAnswerSetsAndCheckStats(solver, 1, 0, 0, 0, 0, 0, 0, 0);
 	}
 
 	@RegressionTest
-	public void checkStatsStringOneChoice(RegressionTestConfig cfg) {
+	public void checkStatsStringOneChoice(SystemConfig cfg) {
 		Solver solver = buildSolverForRegressionTest("a :- not b. b :- not a.", cfg);
 		assumeTrue(solver instanceof StatisticsReportingSolver);
 		collectAnswerSetsAndCheckStats(solver, 2, 1, 1, 1, 1, 0, 0, 0);
 	}
 
-	@RegressionTest
-	public void checkNoGoodCounterStatsByTypeUsingDummyGrounder(RegressionTestConfig cfg) {
-		Solver solver = buildSolverForRegressionTest(atomStore, new GrounderMockWithBasicProgram(atomStore), cfg);
-		assumeTrue(solver instanceof StatisticsReportingSolver);
-		collectAnswerSetsAndCheckNoGoodCounterStatsByType(solver, 4, 0, 0, 0);
-	}
+	// TODO Why are these tests config-dependent if they use a grounder mock and seem rather like unit tests on solver statistics?
```

Comment:
@AntoniusW see `TODO`: I don't see a reason for these tests to be part of this class, i.e. I think these should be unit tests (and not parameterized with different configs). Let me know if you agree and I can move those to an appropriate class in the `alpha-core` module.